### ISSUE TITLE
CMap files are not being loaded from given CMapUrl

### ISSUE THREAD

**Issue by AlexanderSkrock** [wojtekmaj/react-pdf#384]
My pdf-react-component does not show special characters such as ẞ (not ß) and ὠ. As of version 2.3.0 of react-pdf it should be possible to specify parameters to use cmap files to support non-latin characters. Sadly, this does not seem to work properly in my environment.

To me it looks like the cmap-files from my specified url are never loaded or even requested (since the chrome developer tools do not list a request).
While debugging I saw that during runtime an instance of a CMapReaderFactory with specified baseUrl and isCompressed-attribute is created. However, its fetch-method was never called.

For setting up the use of cmaps I followed along the instructions in react-pdf's readme file:
1. I configured the CopyWebpackPlugin to copy the cmaps-folder from pdfjs-dist to the output path of webpack
`new CopyWebpackPlugin([
      {
        from: path.resolve("node_modules/pdfjs-dist/cmaps/"),
        to: "cmaps/"
      }
    ])`

2. I added the cMap-options to my react component. For cMapUrl I put "workplace/cmaps/" because the cmaps-folder is in a subfolder of webpack's static-folder. Since the cmap-files from pdfjs-dist are binary-packed I also added the cMapPacked attribute.

`const cMapUrl = "workplace/cmaps/";
const cMapPacked = true;
const options = {
  cMapUrl,
  cMapPacked
};`

`render() {
    const { numPages, pageScale, actions } = this.state;
    const { file, isDataBase64 } = this.props;

    if (!file) {
      return null;
    }

    return (
      <Box className="pdfContainer" margin="small">
        <Document
          file={ isDataBase64 ? `data:application/pdf;base64,${file}` : file }
          options={ options }
          loading="Das PDF wird geladen..."
          onLoadSuccess={ ::this.onDocumentLoadSuccess }
          className="pdfDocument">
          { _.range(1, numPages + 1).map(page => (
            <Page
              key={ `page_${page}` }
              pageNumber={ page }
              className="pdfPage"
              renderMode="svg"
              scale={ pageScale }
              renderTextLayer={ false }
            />
          )) }
        </Document>
        <Box direction="row" className="actionContainer">
          { actions }
        </Box>
      </Box>
    );`

**Environment:**
- Chrome Version 73.0.3683.86
- react-pdf 4.0.5
- webpack 3.8.1

**Comment by wojtekmaj** [comment]
Hmmm. Does your PDF displays correctly on http://projects.wojtekmaj.pl/react-pdf/test/ ?

**Comment by AlexanderSkrock** [comment]
Yes, I just tried. My PDF  does display correctly on your test page.
Do you have any idea what I could have missed on setting up react-pdf in my environment?

**Comment by Waize** [comment]
Turns out, that our pdf file was wrong. 
Everything works fine now, this issue can be closed :)
